design/APICorrespondance.md

```diff
@@ -0,0 +1,12 @@
+# API Term correspondance
```

Comment:
Nit: Correspondance --> Correspondence 

Be sure and rename the file as well.

Reply:
Done